doc/geany.txt

```diff
@@ -3357,6 +3357,10 @@ Delete to line start            Ctrl-Shift-BackSpace      Deletes from the begin
                                                           current caret position.
 
 Duplicate line or selection     Ctrl-D                    Duplicates the current line or selection.
+							  If the selection spans more than one line,
+							  this duplicates the lines containing selections.
+							  If the selection spans only a single line, then only
+							  the selection is duplicated.
 
```

Comment:
Oh, ok, "Duplicate whole lines, either all lines wholly or partly in the selection, or the current line if no selection."